dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/expand/CuringGlobalParams.java

```diff
@@ -155,16 +154,20 @@ public Map<String, Property> paramParsingPreparation(@NonNull TaskInstance taskI
         // of the process instance complement
         Map<String, String> cmdParam = JSONUtils.toMap(processInstance.getCommandParam());
         String timeZone = cmdParam.get(Constants.SCHEDULE_TIMEZONE);
-        Map<String, String> params = BusinessTimeUtils.getBusinessTime(commandType, scheduleTime, timeZone);
-
-        if (MapUtils.isNotEmpty(globalParamsMap)) {
-            params.putAll(globalParamsMap);
-        }
 
+        // build-in params
```

Comment:
Since we have many build param than before, can we separate to new private method to handle the build-in param?